### PR TITLE
Added User guides for Observability Setup & Comparative Analysis.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

**What this PR does / why we need it**:

**This PR will add user guides -** 
- Observability setup ( Overview Screen )
- workflow analysis
- comparative-analysis of workflows ( Workflows Dashboard screen )

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [X] Signed the commit for DCO to be passed
-   [X] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/user-guides/analyze-workflow.md [hunk 1]
```diff
@@ -4,4 +4,52 @@ title: Analyzing a Workflow
 sidebar_label: Analyzing Workflow
 ---
 
-## Coming soon
+---
+
+Workflow Analytics allows the users to analyze their workflows (Recurring/ non-Recurring) & helps them to keep track of the workflow runs of scheduled workflows and their behaviour.
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/analyze-workflow.md]:
```suggestion
Workflow Analytics allows the users to analyze their workflows (Recurring/ Nonrecurring) & helps them to keep track of the workflow runs of scheduled workflows and their behaviour.
```
It'll be a single word https://www.dictionary.com/browse/nonrecurring#:~:text=adjective,to%20occur%20or%20happen%20again.
If used in other places we can change that.

File: website/docs/user-guides/comparative-analysis.md [hunk 2]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
 sidebar_label: Compare Workflows
 ---
 
-## Coming soon
+Workflow Dashboard allows users to manage their workflows runs, schedules & statistics.
+
+### Linear Area Graph
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]:
```suggestion
### Long-term Statistics
```

File: website/docs/user-guides/comparative-analysis.md [hunk 3]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
...
+<img src={require('../assets/schedules-stats.png').default} />
+<img src={require('../assets/schedules-stats-dropdown.png').default} />
+
+:::info
+
+Users can hover over the respective graphs to get the total workflow runs & total schedules respectively.
+
+:::
+
+### Workflow Runs Radial chart
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]:
```suggestion
### Total Workflow Statistics
```

File: website/docs/user-guides/comparative-analysis.md [hunk 4]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
...
+
+Workflows runs radial chart allows users to see how many of their scheduled have been passing/ failing & running. This helps you to see average behaviour of all the workflow runs.
+
+<img src={require('../assets/workflows-radial-chart.png').default} />
+
+### Workflows Comparison
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]:
```suggestion
### Comparing Multiple Workflows
```

File: website/docs/user-guides/observability-set-up.md [hunk 5]
```diff
@@ -4,4 +4,49 @@ title: Setup Observability with Chaos Center
...
+
+After setting up the dashboard, users will be able to see the dashboard on the overview page.
+
+<img src={require('../assets/overview-with-dashboards.png').default} />
+
+Users can view, configure & download the dashboards in the form of JSON for reusability.
+
+You can check the workflow analytics and the workflow run graph by clicking on _**see workflow run**_ & _**see workflow analytics**_ icons for a workflow from the list of workflow dashboards present on the overview page.
+
+## Learn More
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/observability-set-up.md:52]: Link to `Analysing Workflows` , `Manage Application Dashboard` and `Configure Datasource`

File: website/docs/user-guides/observability-set-up.md [hunk 6]
```diff
@@ -4,4 +4,49 @@ title: Setup Observability with Chaos Center
...
+### Overview
+
+Overview Tab helps you to manage your connected data sources and dashboards, all while making it easier to view and analyze your workflows. The overview tab is split into 3 parts -
+
+- Data Sources
+- Application Dashboards
+- Workflow Dashboards
+
+:::note
+When there is no agent connected or in the `active` state, the overview tab will show a banner for connecting an agent.
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/observability-set-up.md]: This part of connecting an Agent can be moved to `Before you Begin` section and can be linked to Litmusctl connecting and agent page. Also, the image is not required as the UX is very clear here.

File: website/docs/user-guides/analyze-workflow.md [hunk 7]
```diff
@@ -4,4 +4,52 @@ title: Analyzing a Workflow
...
+
+:::
+
+### Experiments Table
+
+On clicking on a bar, more details about the workflow run represented by that stacked bar are presented in a tabular format. A table having individual experiments details is presented to the user, which helps them to get insights about the experiments.
+
+<img src={require('../assets/experiments-table.png').default} />
+
+## Learn More
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/analyze-workflow.md:55]: Link to `Compare Workflows`, `Manage Application Dashboard and `Configure Datasource`

File: website/docs/user-guides/comparative-analysis.md [hunk 8]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
...
+
+The Comparison report can be downloaded in PDF form using the _**export button**_ given in header of table.
+
+<img src={require('../assets/export-button.png').default} />
+
+> **Sample workflows comparison report can be downloaded from [here](../assets/chaos-center-analytics.pdf)**
+
+## Learn More
+
+- [See Workflow Analytics](analyze-workflow)
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md:70]: Add two more links to `Manage Application Dashboard and `Configure Datasource`

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: website/docs/user-guides/comparative-analysis.md [hunk 9]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
 sidebar_label: Compare Workflows
 ---
 
-## Coming soon
+Workflow Dashboard allows users to manage their workflows runs, schedules & statistics.
+
+### Linear Area Graph
+
+Linear Area graph gives the statistics of workflow runs & statistics.
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]: change language

File: website/docs/user-guides/comparative-analysis.md [hunk 10]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
...
+
+:::info
+
+Users can hover over the respective graphs to get the total workflow runs & total schedules respectively.
+
+:::
+
+### Workflow Runs Radial chart
+
+Workflows runs radial chart allows users to see how many of their scheduled have been passing/ failing & running. This helps you to see average behaviour of all the workflow runs.
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]: change language

File: website/docs/user-guides/comparative-analysis.md [hunk 11]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
...
+
+<img src={require('../assets/workflows-radial-chart.png').default} />
+
+### Workflows Comparison
+
+The workflows table helps the users to keep track of their scheduled workflows along with associated details like workflow name, target agent, starting time,etc & allows to compare the workflows by selecting them from the table.
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]: change language

File: website/docs/user-guides/comparative-analysis.md [hunk 12]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
...
+
+<img src={require('../assets/workflows-comparison-table-1.png').default} />
+
+For comparing the workflows, Users can select the workflows from the table & click on _**Compare workflows**_ button.
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]:
```suggestion
For comparing the workflows, users can select the workflows from the table & click on _**Compare workflows**_ button.
```

File: website/docs/user-guides/comparative-analysis.md [hunk 13]
```diff
@@ -4,4 +4,66 @@ title: Comparative Analysis of Chaos Workflows
...
+
+<img src={require('../assets/workflows-comparison-table-2.png').default} />
+
+Once, User clicks on _**Compare workflows**_ button, they will be presented with a graph, which shows comparison of selected workflows with respect to resilience score.
```
Comment (arkajyotiMukherjee) [inline website/docs/user-guides/comparative-analysis.md]:
```suggestion
Once, user clicks on _**Compare workflows**_ button, they will be presented with a graph, which shows comparison of selected workflows with respect to resilience score.
```

File: website/docs/user-guides/observability-set-up.md (hunk 6 above)
Comment (arkajyotiMukherjee) [reply website/docs/user-guides/observability-set-up.md]: change language

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Comment by vanshBhatia-A4k9** [comment]
Thank you for the additions :100:

**Pull request merged**